bitnami/minio/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 6.5.0
+version: 6.6.0
```

Comment:
Could you please replace it with a PATH version bump? (6.5.1)

We use [semver](https://semver.org/) and this would be a PATCH version as it is a backward-compatible bug fix and not a new functionality.